src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -2366,8 +2366,8 @@ def test_sql_tdebyok(self, resource_group, server):
 
 
 class SqlServerVnetMgmtScenarioTest(ScenarioTest):
-    @ResourceGroupPreparer()
-    @SqlServerPreparer()
+    @ResourceGroupPreparer(location='eastus')
+    @SqlServerPreparer(location='eastus')
```

Comment:
why add location parameters here?

Reply:
Service made changes without notifying us, the default lcoation doesn't allow to create SQL server (with our sub?).
I tested some locations, eastus is available.